webapp/src/pages/eventHome/EventHome.js

```diff
@@ -49,7 +49,7 @@ class EventInfo extends Component {
         <h2>{event.description}</h2>
         <EventStatus longForm={true} event={event} />
         {isEventAdmin(this.props.user, this.props.event) 
-        && <EventStats event={this.props.event}/>}
+        && <EventStats submissionLink={false} event={this.props.event}/>}
```

Comment:
Did you mean to pass this to the previous one, `EventStatus` ?

Reply:
Yes, but I believe we do not even need to pass this when longForm is true.